2200033_박지환/문제2.py

```diff
@@ -0,0 +1,21 @@
+# 신용카드 만들기 1
+
+T = int(input())
+
+card_list = [list(map(int, input().split())) for i in range(T)]
+
+
+for i in range(T):
+    sum = 0
+    for j in range(len(card_list[i])): 
+        if (j+1) % 2 == 1: # 카드 리스트의 홀수 번째는 2 곱해서 더함
+            sum += card_list[i][j]*2
+        elif (j+1) % 2 == 0:  # 카드 리스트의 짝수 번째는 그대로 더함
+            sum += card_list[i][j]
+
+    print(f'#{i+1} {abs((sum % 10)-10)%10}') # 다 더한 값을 1. 10으로 나눈 나머지에 
```

Comment:
```suggestion
    print(f'#{i+1} {(10 - (sum%10))%10}')
```
10을 나눈 나머지에 10을 빼면 항상 0보다 작거나 같으므로 절댓값 함수를 사용하지 않고 답을 출력할 수 있습니다! 👍

Reply:
(sum %10) 후에 10- 0 = 10 의 경우에서 어떻게 해야 할지 모르겠어서 복잡하게 풀었는데, %10을 한번 더 하면 되었군요! 감사합니다 ㅎㅎ